override-cp.yaml

```diff
@@ -1,10 +1,10 @@
-from: "redhat-sso-7/sso72:cp"
+from: "redhat-sso-7/sso72:latest"
```

Comment:
@slaskawi This is not needed (looks the `cp` tag on the standalone image is set based on `container.yaml` file depending in which branch the standalone image was built).

Reply:
Fixed, thanks!